code/game/objects/items/crayons.dm

```diff
@@ -190,7 +190,7 @@
 	..()
 
 /obj/item/toy/crayon/mime/Topic(href,href_list)
-	if((usr.restrained() || usr.stat || usr.is_in_active_hand(src)))

```

Comment:
Eh, if they drop the crayon it doesn't make sense that they can change the crayon colour, right?